sdk/computefleet/azure-resourcemanager-computefleet/src/test/java/com/azure/resourcemanager/computefleet/ComputeFleetManagerTests.java

```diff
@@ -76,20 +77,22 @@ public void beforeTest() {
 
         computeFleetManager = ComputeFleetManager
             .configure()
-            .withLogOptions(new HttpLogOptions().setLogLevel(HttpLogDetailLevel.BASIC))
+            .withLogOptions(new HttpLogOptions().setLogLevel(HttpLogDetailLevel.BODY_AND_HEADERS))
```

Comment:
Please switch back to `BASIC` before merge.

Reply:
Fixed in the new version.